lib/Dancer/Plugin/SimpleCRUD.pm

```diff
@@ -219,6 +227,35 @@ which returns the WHERE clause hashref - for instance:
 
   where_filter => sub { { customer_id => logged_in_user()->{customer_id} } },
 
+=item <joins> (optional)
+
+Specify one more more joins to other tables in order to fetch additional data for each
+row to be displayed. Useful with custom_columns. For example:
+
+    simple_crud(
+        prefix=>'bar_with_join',
+        db_table=>"users",
+        joins => [
+            {   # join with the 'user_extras' table and select 'select_columns' 
+                # where the table.id=join_table.join_column
+                table=>"user_extras",   
+                join_style=>"join",     # "join" is default. "left join" ok too.
+                select_columns=>["extra"], 
+                key_column=>'id',
+                join_column=>'user_id',
+            },
```

Comment:
I don't like this design because with foreign_keys, key_column matches the right table, and with joins, key_column matches the left table. Reconsidering.